backend/seed_data/units.yaml

```
@@ -0,0 +1,8 @@
+units:
+  - title: "Hash Maps"
+    order: 1
+    course_id: 1
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)